drift.py

```diff
@@ -1,140 +1,77 @@
 from helicase import Helicase
-from datetime import *
-import argparse
-import json, requests
+from datetime import datetime
+from spack import *
+import json, requests, argparse
```

Comment:
This is a nit, but most libraries I think it's best practice to import entire modules on different lines, e.g.

```
import argparse
import json
import requests
```
Also notice in abc order! You can check out isort that will nitpick this for you :)

Reply:
Oh perfect! I'm definitely forgetting my python-isms! And thanks for the tool suggestion!